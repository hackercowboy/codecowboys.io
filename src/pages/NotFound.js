import PropTypes from 'prop-types';
import React from 'react';
import { injectIntl } from 'react-intl';

import Container from '../components/Container';
import Main from '../components/Main';
import Navigation from '../components/Navigation';

function NotFound({ intl }) {
  return (
    <Main>
      <Navigation />
      <Container>
        <h1 style={{ padding: '20px 0' }}>{intl.formatMessage({ id: 'lost.title' })}</h1>
        <p className="lead">{intl.formatMessage({ id: 'lost.description' })}</p>
        <p className="flex-grow"><a className="btn btn-primary" href="https://codecowboys.io">{intl.formatMessage({ id: 'lost.button' })}</a></p>
      </Container>
    </Main>
  );
}

NotFound.propTypes = {
  intl: PropTypes.shape({ formatMessage: PropTypes.func.isRequired }).isRequired,
};

export default injectIntl(NotFound);
