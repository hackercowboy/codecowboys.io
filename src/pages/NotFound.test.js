import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

describe('<NotFound/>', () => {
  it('should render', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
