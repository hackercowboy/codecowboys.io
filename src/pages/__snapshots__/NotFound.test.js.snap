// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound/> should render 1`] = `
"<Main className={[undefined]}>
  <Navigation />
  <Container>
    <h1 style={{...}}>
      You&#39;re Lost! 404 - Not Found
    </h1>
    <p className=\\"lead\\">
      The page your are looking for doesn&#39;t exist
    </p>
    <p className=\\"flex-grow\\">
      <a className=\\"btn btn-primary\\" href=\\"https://codecowboys.io\\">
        Go Home
      </a>
    </p>
  </Container>
</Main>"
`;
